bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -5,4 +5,6 @@
 	<type>binding</type>
 	<name>TapoControl Binding</name>
 	<description>Control your TAPO-SmartHome Devices</description>
+	<connection>cloud</connection>
```

Comment:
I'm wondering if this should be `cloudDiscovery`? Reference: https://next.openhab.org/docs/developer/addons/addon.html

Reply:
That's a question of definition.

> cloudDiscovery for add-ons that require a cloud connection for set-up

Cloud conncection is required for get credentials to login to devices. So i think it could be "cloud"

Reply:
Thanks for the clarification. I didn't understand from the documentation that cloud is a requirement for using the binding after discovery:
- https://www.openhab.org/addons/bindings/tapocontrol/#prerequisites
- https://www.openhab.org/addons/bindings/tapocontrol/#discovery
- https://www.openhab.org/addons/bindings/tapocontrol/#bridge-configuration